contracts/StableJoeStaking.sol

```diff
@@ -224,17 +231,19 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
     function pendingReward(address _user, IERC20Upgradeable _token) external view returns (uint256) {
         require(isRewardToken[_token], "StableJoeStaking: wrong reward token");
         UserInfo storage user = userInfo[_user];
-        uint256 _totalJoe = internalJoeBalance;
```

Comment:
Here it's not important as it's a view function, external it will be free either way